internal/removals/remove.go

```diff
@@ -164,6 +174,28 @@ func GetRemainingMap(logger logr.Logger, adminClient fdbadminclient.AdminClient,
 	return remainingMap, nil
 }
 
+// verifyExcludedProcesses will verify that the processes that should be excluded based on the FoundationDB machine-readble status are also excluded when issuing
```

Comment:
> verifyExcludedProcesses will verify that the processes that should be excluded based on the FoundationDB machine-readble status

We are verifying because we don't trust the exclude information provided by machine-readable status? Or the exclude information provided by machine-readable status is not meant to be precise?

Reply:
We want to have an additional safeguard in cases where the machine-readable status might be missing some information like the role information for processes.